rest/test/db/CatapultDb_spec.js

```diff
@@ -443,6 +443,42 @@ describe('catapult db', () => {
 				block => expect(block).to.deep.equal(stripExtraneousBlockInformation(renameId(seedBlock)))
 			);
 		});
+
+		it('can retrieve blocks with transactions at heights', () => {
```

Comment:
consider grouping these in own describe

Reply:
consider swapping order of tests so they go from simpler to more complex

Reply:
Added describe, a new projection unit test and resorted tests.